src/ussd_popi_rapidpro.js

```diff
@@ -1532,105 +1651,6 @@ go.app = function() {
             });
         });
 
-        self.add("state_all_questions_view", function(name) {
```

Comment:
Why are these states being removed?

Reply:
@erikh360 I've removed them as we don't have them on the miro flow. Must i add them back ?